src/OrchardCore.Modules/OrchardCore.Users/Controllers/EmailAuthenticatorController.cs

```diff
@@ -170,15 +166,10 @@ public async Task<IActionResult> SendCode()
         var settings = (await SiteService.GetSiteSettingsAsync()).As<EmailAuthenticatorLoginSettings>();
         var code = await UserManager.GenerateTwoFactorTokenAsync(user, TokenOptions.DefaultEmailProvider);
 
-        var message = new MailMessage()
-        {
-            To = await UserManager.GetEmailAsync(user),
-            Subject = await GetSubjectAsync(settings, user, code),
-            Body = await GetBodyAsync(settings, user, code),
-            IsHtmlBody = true,
-        };
-
-        var result = await _emailService.SendAsync(message);
+        var to = await UserManager.GetEmailAsync(user);
```

Comment:
I think the old version was nicer lol

Reply:
I'm not sure why there's two async version, I might change this before I merge this PR